test/lib/cross_agent_tests/utilization_vendor_specific/aws.json

```diff
@@ -4,8 +4,8 @@
     "uri": {
       "http://169.254.169.254/latest/dynamic/instance-identity/document": {
         "response": {
-          "instanceId": null,
-          "instanceType": null,
+          "instanceId": null,      
```

Comment:
It shouldn't matter with json, but are we adding line-ending whitespace here?

Reply:
I didn't touch this file. It must've been from syncing with other repo